jep/0000/README.adoc

```diff
@@ -0,0 +1,161 @@
+= JEP-0000: Platform Special Interest Group
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 0000
+
+| Title
+| Platform Special Interest Group (Platform SiG)
+
+| Sponsor
+| link:https://github.com/oleg-nenashev[Oleg Nenashev]
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Informational
+
+| Created
+| 2018-07-04
+
+| BDFL-Delegate
+| :bulb: Link to github user page :bulb:
+
+| JIRA
+| link:https://issues.jenkins-ci.org/browse/INFRA-1701[INFRA-1701]
+
+| Discussions-To
+| https://groups.google.com/forum/#!topic/jenkinsci-dev/YNZ3DWGY824
+//
+//
+// Uncomment if this JEP depends on one or more other JEPs.
+//| Requires
+//| :bulb: JEP-NUMBER, JEP-NUMBER... :bulb:
+//
+//
+// Uncomment and fill if this JEP is rendered obsolete by a later JEP
+//| Superseded-By
+//| :bulb: JEP-NUMBER :bulb:
+//
+//
+// Uncomment when this JEP status is set to Accepted, Rejected or Withdrawn.
+//| Resolution
+//| :bulb: Link to relevant post in the jenkinsci-dev@ mailing list archives :bulb:
+
+|===
+
+## Abstract
+
+This informational Jenkins Enhancement Proposal describes the Platform Special Interest Group,
+which has been proposed as a part of the Jenkins & Java 10+ Hackathon.
+The Group offers a place for all kinds of platform support discussions:
+Java, Operating Systems, Architectures, Docker, Packaging, Web Containers, etc.
+
+The JEP is documented according to the link:https://groups.google.com/forum/#!msg/jenkinsci-dev/6-1mZoKp4hM/yEria93ZAAAJ[SiG proposal in jenkinsci-dev]
+by link:http://github.com/rtyler[R. Tyler Croy].
+
+## Specification
+
+### Domain/focus area
+
+* Platform SiG focuses on technologies used in Jenkins: Java, Operating Systems, Architectures, Packaging, Web Containers
+* Platform SiG does not focus on Cloud-Native Platforms, there is a separate Cloud-Native SiG for that (JEP-TODO)
+
+Examples of topics:
+
+* Defining platform support policies (e.g. “defining Windows support policy”)
+* Coordinating effort on new platform support (e.g. Java 10 in JEP-TODO)
+* Working with external communities on better platform support and packaging (e.g. improving support of IBM Java, enabling OpenIndiana packaging, ARM Support in Jenkins, adapting RedHat packaging to best practices, etc.)
+* Reviewing JEPs submitted in the area
+
+### Meetings
+
+There will be regular SiG meetings scheduled.
+Initially the meetings will be conducted once per month,
+then they may be adapted according to the activity.
+
+Meetings will be conducted and recorded via Jenkins Hangouts-on-Air.
+
+### Chairs
+
+* Oleg Nenashev
+* TBD
+
+### Channels
+
+* `jenkins-platform-sig` Google group for internal sync-ups
```

Comment:
Since the Google group can be represented as a URL, could this be changed to

> * `link:https://groups.google.com/forum/#!forum/jenkins-platform-sig[jenkins-platform-sig]` Google group for internal sync-ups